department-of-veterans-affairs/va.gov-cms
# VACMS-2551: Q&A drag and drop UX

## Description
See #2552 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/90407422-83c86d00-e074-11ea-9b62-12e1488a39ce.png)

## QA steps
As an admin, go to `/node/add/faq_multiple_q_a` after adding 2 published Q&A nodes. Add them to the `Q&A GROUPS` fieldset, and visually verify that they are in a horizontally stacked, draggable display.
